streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskExecutor.java

```diff
@@ -110,11 +120,11 @@ private long processTask(final Task task, final int maxNumRecords, final Time ti
                 "Will trigger a new rebalance and close all tasks as zombies together.", task.id());
             throw e;
         } catch (final StreamsException e) {
-            log.error("Failed to process stream task {} due to the following error:", task.id(), e);
+            log.error(String.format("Failed to process stream task %s due to the following error:", task.id()), e);
```

Comment:
Sorry to see log4j still have not figured out the way for both string param and exception in presentation..